### DISCUSSION TITLE
How to get only Q1 laps data?

### DISCUSSION THREAD

**Discussion by harish5p** [theOehrly/Fast-F1#202] [Q&A]
`Session = fastf1.get_session(year, race, session)
    laps = Session.load_laps(with_telemetry=True)`

If I use 'Time' column to filter, whenever we have red flags, quali session gets delayed, so it becomes difficult to standardize . 

After loading laps data, how to keep only Q1 laps?

**Comment by theOehrly** [comment]
There currently is no built-in way to do this. Although, it is planned to be added at some point.
You can likely use the data in `Session.session_status` as a reference. (see https://theoehrly.github.io/Fast-F1/core.html#fastf1.core.Session.session_status)
